lib/workos/audit_logs.rb

```diff
@@ -57,10 +59,13 @@ def create_event(organization:, event:, idempotency_key: nil)
           range_end: String,
           actions: T.nilable(T::Array[String]),
           actors: T.nilable(T::Array[String]),
+          actor_names: T.nilable(T::Array[String]),
+          actor_ids: T.nilable(T::Array[String]),
           targets: T.nilable(T::Array[String]),
         ).returns(WorkOS::AuditLogExport)
       end
-      def create_export(organization:, range_start:, range_end:, actions: nil, actors: nil, targets: nil)
+      def create_export(organization:, range_start:, range_end:, actions: nil, # rubocop:disable Metrics/ParameterLists
```

Comment:
I think it should be backwards compatible since these are keyword arguments (there's no way to use them positionally afaik):
```ruby
irb(main):001:0> def foo(a:, b:); [a, b]; end
=> :foo
irb(main):002:0> foo(b: "b", a: "a")
=> ["a", "b"]
irb(main):003:0> foo(a: "a", b: "b")
=> ["a", "b"]
irb(main):004:0> foo("a", "b")
(irb):1:in `foo': wrong number of arguments (given 2, expected 0; required keywords: a, b) (ArgumentError)
```

Reply:
(still happy to swap the order though if that's preferred)